Classes/BITUpdateManager.m

```diff
@@ -727,10 +727,16 @@ - (BOOL)initiateAppDownload {
   
   NSString *hockeyAPIURL = [NSString stringWithFormat:@"%@api/2/apps/%@?format=plist%@", self.serverURL, [self encodedAppIdentifier], extraParameter];
   NSString *iOSUpdateURL = [NSString stringWithFormat:@"itms-services://?action=download-manifest&url=%@", bit_URLEncodedString(hockeyAPIURL)];
-  
+
+  // Notify delegate of update intent before placing the call
+  if (self.delegate != nil && [self.delegate respondsToSelector:@selector(willStartDownloadAndUpdate:)]) {
```

Comment:
The `self.delegate != nil` is unnecessary; `[nil respondsToSelector:]` return NO.